advanced/Scripts/webpage.sh

```diff
@@ -729,7 +729,12 @@ AddCustomCNAMERecord() {
     if [[ -n "${validDomain}" ]]; then
         validTarget="$(checkDomain "${target}")"
         if [[ -n "${validTarget}" ]]; then
-            echo "cname=${validDomain},${validTarget}" >> "${dnscustomcnamefile}"
+            if [ "${validDomain}" = "${validTarget}" ]; then
+                echo "  ${CROSS} Domain and target are the same!"
```

Comment:
Shouldn't we print a small info text about why this is a bad thing?

Like
```suggestion
                echo "  ${CROSS} Domain and target are the same. Not adding to prevent looping."
```
(but maybe better)

Reply:
Maybe not better, but different?